### PR TITLE
Follow command and ahelp menu follow

### LINKED ISSUE #7969: AHelp and verb "teleport to" functionality should use following
## Description

If you're following someone and tp to someone else it breaks since you're still parented to the other person. IMO  the ahelp button should just use following instead like what ghost roles do

The verbs should also use following (at least TP to should obv)

The actual command is in engine so can't change that, but the content stuff can be changed to just use something else

**Screenshots**
these

image

image

**Additional context**

### PR DESCRIPTION
**Pull request by LankLTE**
## About the PR

Command "follow [netEnt]" to begin orbiting an entity. Also replaces the "Teleport to" button on the ahelp menu with a "follow" button. 
image

The follow command doesn't check that the entity is a ghost, but you can do some funny admeme stuff with that (like making a plushie follow a player or whatever) so it should be fine. 

## Why / Balance

Resolves #7969. The command was necessary to make this work but should probably exist anyways. 

## Technical details

New command simply makes sure both entities exists then makes the user follow the target. Doesn't check if the person is a ghost hence requiring admin perms. 

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

Let me know if I fucked up the admin CL

:cl: Lank
ADMIN:
- add: Added a new "follow" command. 
- tweak: Replaced the ahelp menu's "Teleport to" verb with "Follow".

### CODE REVIEW

**Review by metalgearsloth — DISMISSED** [review]
Command needs:
- filescoped namespace
- Localisation
- Use WriteError not WriteLine.

**Comment by LankLTE** [comment]
> Command needs:

should be good now

**Comment by mirrorcult** [comment]
is there a reason the follow command is adminonly? i think it'd be better if the command worked always but just enforced checks if the caller isn't an admin, so you can do stuff like bind a key to follow enthover

**Comment by LankLTE** [comment]
> is there a reason the follow command is adminonly? i think it'd be better if the command worked always but just enforced checks if the caller isn't an admin, so you can do stuff like bind a key to follow enthover

Following some entities causes a fuck ton of errors for various reasons. Given admins can already basically crash the server I don't see it as a huge issue but giving players that capability is bad and I have no idea how to check for that reliably.

**Comment by mirrorcult** [comment]
thats kind of a silly reason but i suppose its fine for now

File: Content.Client/Administration/UI/Bwoink/BwoinkControl.xaml.cs [hunk 1]
```diff
@@ -204,8 +204,8 @@ public void UpdateButtons()
             Respawn.Visible = _adminManager.CanCommand("respawn");
             Respawn.Disabled = !Respawn.Visible || disabled;
 
-            Teleport.Visible = _adminManager.CanCommand("tpto");
-            Teleport.Disabled = !Teleport.Visible || disabled;
+            Follow.Visible = _adminManager.CanCommand("tpto");
```
Comment (mirrorcult) [inline Content.Client/Administration/UI/Bwoink/BwoinkControl.xaml.cs:207]: shouldnt this be `CanCommand("follow")`

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Content.Client/Administration/UI/Bwoink/BwoinkControl.xaml.cs (hunk 1 above)
Comment (LankLTE) [reply Content.Client/Administration/UI/Bwoink/BwoinkControl.xaml.cs:207]: yes it should be, I missed that bit when renaming it

**Comment by LankLTE** [comment]
minor error was fixed, should be good now

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**